docs/dotnet/dotnet-http-metrics.md

```diff
@@ -0,0 +1,238 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: HTTP client and server
+--->
+
+# Semantic Conventions for HTTP client and server metrics emitted by .NET
+
+**Status**: [Experimental, Feature-freeze][DocumentStatus]
+
+This article defines semantic conventions for HTTP metrics emitted by .NET components and runtime.
+
+<!-- toc -->
+
+- [HTTP client](#http-client)
+  * [Metric: `http.client.request.duration`](#metric-httpclientrequestduration)
+  * [Metric: `http.client.open_connections`](#metric-httpclientopen_connections)
+  * [Metric: `http.client.connection.duration`](#metric-httpclientconnectionduration)
+  * [Metric: `http.client.request.time_in_queue`](#metric-httpclientrequesttime_in_queue)
+  * [Metric: `http.client.active_requests`](#metric-httpclientactive_requests)
+- [HTTP server](#http-server)
+  * [Metric: `http.server.request.duration`](#metric-httpserverrequestduration)
+  * [Metric: `http.server.active_requests`](#metric-httpserveractive_requests)
+
+<!-- tocstop -->
+
+## HTTP client
+
+All Http client metrics are reported by the `System.Net.Http` meter.
+
+### Metric: `http.client.request.duration`
+
+Client request duration measures the time it takes to receive response headers and doesn't include time to read the response body.
+
+This metric follows the common [http.client.request.duration](../http/http-metrics.md#metric-httpclientrequestduration) definition.
+
```

Comment:
```suggestion

This metric SHOULD be specified with
[`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/metrics/api.md#instrument-advisory-parameters)
of `[ 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 ]`.

```

Reply:
this is a standard metric and we just say that it follows common semantics (and list what's different). 

Since it's the same boundaries as on a standard one, let's not mention them.